framework/lib/components/createable-select-dropdown/createable-select-dropdown.tsx

```diff
@@ -1,12 +1,13 @@
-import React, { useMemo, useRef, useState } from 'react'
-import { ActionMeta, CreatableSelect } from 'chakra-react-select'
-import type { Option, SingleValue } from '@northlight/ui'
+import React, { useRef, useState } from 'react'
+import { ActionMeta, CreatableSelect, SingleValue } from 'chakra-react-select'
 import { PlusSolid } from '@northlight/icons'
-import { Box, Icon, useOutsideClick } from '@chakra-ui/react'
-import { isNil } from 'ramda'
+import { Box, Icon } from '@chakra-ui/react'
+import R, { isEmpty, not, trim } from 'ramda'
```

Comment:
In most cases, we don't import `R` namespace. Try to utilize direct imports only,
such as `import { isEmpty, not, trim }` from 'ramda'